# Object label ID
int16 label_id

# Object closest position
float32[3] position

# Object velocity
float32[3] velocity

# 3D Person skeleton in world frame
zed_yolo_ros/Skeleton3D skeleton_3d
